dashboard/views.py

```diff
@@ -27,12 +29,31 @@
 GRADE_LOW="low_grade"
 NO_GRADE_STRING = "NO_GRADE"
 
+EVENT_VIEW_FILE_ACCESS="VIEW_FILE_ACCESS"
```

Comment:
These constants don't seem to add much value since they're only used once. I wonder if there's a better practice for this. 

Maybe a "little" better organization might be to put this in a class like 

```
class Events:
    VIEW_FILE_ACCESS = "VIEW_FILE_ACCESS"
    VIEW_ASSIGNMENT_PLANNING = "VIEW_ASSIGNMENT_PLANNING"
    VIEW_GRADE_DISTRIBUTION = "VIEW_GRADE_DISTRIBUTION"

```
Then you'd call Events.VIEW_GRADE_DISTRIBUTION

Just a thought, I don't know what's better. Probably both are about the same at the moment. ;)

Reply:
I will keep that in mind. Maybe not changing this time. Will do that once we have more events.

Reply:
Yeah, I think once we get more events we could have the EVENT_ID as well as the data payload all defined inside the class. So all of these are better together. Would be nice as this event model expands out. A good idea for future refactor as we add more for sure.